examples/misc/rasterization_demo.py

```diff
@@ -4,50 +4,57 @@
 ==================
 
 """
+
+# Rasterization is a method where an image described in a vector graphics format is being converted into a raster image (pixels).  
```

Comment:
Can you add a sentence about how we can flag particular parts of a Figure to be rasterized and some comments about why you would want to do this?  Something like:


This demo is showing how you may want to balance most things (text, lines, ...) being vectorized (where you get all of the benefits of vector graphics), but turning some of the artists into rasterized images (because embedding a 2k by 2k mesh in an SVG results in a huge file that takes a long time to both save and load).

?